app/models/cache/Answers.scala

```diff
@@ -51,7 +51,7 @@ object ArrivalAnswers {
 }
 
 case class AssociateUcrAnswers(
-  override val eori: Option[String] = None,
+  override val eori: Option[String] = Answers.fakeEORI,
```

Comment:
Does this work?  Is the fake eori persisted/submitted at the end of the journey?
I think I tried something like this and it didn't work

Reply:
The same like passing in Choice Controller as arguments

Reply:
OK - I trust you!